itou/static/js/modal_control.js

```diff
@@ -0,0 +1,25 @@
+$(document).ready(function(){
+
+    /**
+     * JS that can be used in combination with HX-Trigger to show/hide modals
+     * via an HTMX response:
+     * "HX-Trigger": {"modalControl": {"id": "delete_prior_action_modal", "action": "hide"}}
+     **/
+
+    document.body.addEventListener("modalControl", function(evt){
+        if (!evt.detail.id) {
+            console.log("Received modalControl event without id")
+            return
+        }
+        if (!evt.detail.action) {
+            console.log("Received modalControl event without action")
+            return
+        }
+        if (!['show', 'hide', 'toggle'].includes(evt.detail.action) ) {
```

Comment:
Le cas d’utilisation de `toggle` n’est pas clair, autant ne pas le supporter jusqu’à ce qu’on en ait besoin.
```suggestion
        if (!['show', 'hide'].includes(evt.detail.action) ) {
```